packages/components/src/Menu/Menu.module.css

```
@@ -65,24 +65,26 @@
 
 .action {
   display: flex;
-  gap: var(--menu-space);
   width: 100%;
   padding: var(--menu-space);
   border: none;
   border-radius: var(--radius-base);
+  outline: transparent;
   text-align: start;
   background-color: transparent;
   cursor: pointer;
+  gap: var(--menu-space);
   align-items: center;
+  transition: background-color var(--timing-base) ease-in;
```

Eases the background color change instead of the previous instant flicker

---

I like it. But can I suggest adding the transition to box-shadow as well?
```suggestion
    transition: background-color var(--timing-base) ease-in, box-shadow var(--timing-base) ease-in;
```
Or `transition: all ... `.
It creates a slightly more visually pleasant transition.

https://github.com/user-attachments/assets/cc16052e-d80d-4975-a4bc-0e78573046c9